packages/concerto-core/lib/rootmodel.js

```diff
@@ -32,7 +32,10 @@ function getRootModel(versioned) {
     abstract concept Participant identified {}
     abstract concept Transaction {}
     abstract concept Event {}
-    `;
+    abstract concept Decorator {}
+    concept DotNetNamespace extends Decorator {
```

Comment:
Does `DotNetNamespace` belong here? Perhaps in another namespace?

Reply:
Moved to new ns.